.github/workflows/dev-ci-cd.yml

```
@@ -44,14 +44,24 @@ jobs:
         name: 'Dependabot'
         needs: [build]
         runs-on: ubuntu-latest
+        permissions: # Add required permissions
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)